card/mod/core/set/all/export.rb

```diff
@@ -1,7 +1,7 @@
 format :json do
   before :export do
-    @export_count ? @export_count += 1 : @export_count = 1
-    @exported_keys ||= ::Set.new
+    @export_count = inherit(:export_count).to_i + 1
```

Comment:
I still think renaming this to export_depth would make it easier to understand in the future.  (it's not the total count of exports)